.release/security-scan.hcl

```diff
@@ -5,8 +5,8 @@ container {
 }
 
 binary {
-	secrets      = true
-	go_modules   = true
+	secrets      = false
```

Comment:
Was this on purpose? 🤔

Reply:
I saw a bunch of non-ui changes when I clicked "changes from last review" but I didn't see them on the overall changes so not sure what happened here but just ignore probably

Reply:
Could have been when I merged main in? Otherwise I don't know 😬